packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -139,6 +139,9 @@ class CupertinoTextField extends StatefulWidget {
   /// this is a single-line text field and will scroll horizontally when
   /// overflown. [maxLines] must not be zero.
   ///
+  /// The text cursor is not shown if [showCursor] is false or if [showCursor]
```

Comment:
Could `showCursor` then be false by default instead of null or false?

Reply:
the requirement is showCursor will default to !readonly if user does not specify a value. If i default the input parameter to false, I can't tell if it user pass in or not.